### PR TITLE
Feature/profile refactoring: PR #1 Cosmetic changes

### LINKED ISSUE #1571: Refactor profile
[comment]: # (Please replace ... with your information. Remove < and >)

### User Story

As a developer, I want to refactor the profiles according to #1495 to improve developer experience.

### Description

Currently the structure of the profile directory looks like the following:
```
├── db.cljs
├── edit
│   └── views.cljs
├── events.cljs
├── photo_capture
│   ├── styles.cljs
│   └── views.cljs
├── qr_code
│   ├── styles.cljs
│   └── views.cljs
├── styles.cljs
├── validations.cljs
└── views.cljs
```
According to #1495 it should be:
```
├── edit
│   └── screen.cljs
├── handlers.cljs
├── photo_capture
│   ├── screen.cljs
│   └── styles.cljs
├── qr_code
│   ├── screen.cljs
│   └── styles.cljs
├── screen.cljs
├── specs.cljs
├── styles.cljs
└── validations.cljs
```
1. add letsubs macro in views
2. apply code convention for views (#1495)
3. move nav/preload-data! to navigation.cljs
4. rewrite events using fx and cofx
5. apply code convention for events (#1495)
6. rewrite subs using re-frame best practice (db can be used only in extraction subscriptions)
7. use ns keys in app-db
8. write specs
9. write tests

### PR DESCRIPTION
**Pull request by yenda**
First step of refactoring guidelines to for #1571 

If you use aggressive indent-mode in emacs add this to your .emacs so that it respects the clojure style guide regarding indentation: `(put-clojure-indent 'create-class 0)`

Steps to test:
- play around with profile editing and contact profile to see if anything broke (though is this only a cosmetic PR so nothing should have break)

### CODE REVIEW

File: src/status_im/profile/edit/views.cljs [hunk 1]
```diff
@@ -1,38 +1,30 @@
...
+            [status-im.components.toolbar-new.view :refer [toolbar]]
+            [status-im.i18n :refer [label]]
+            [status-im.profile.styles :as styles]
+            [status-im.profile.validations :as validations]
+            [status-im.profile.views :refer [colorize-status-hashtags]]
+            [status-im.utils.utils :as utils :refer [clean-text]])
+  (:require-macros [status-im.utils.views :refer [defview]]))
 
-(defn edit-my-profile-toolbartoolbar []
+(defn edit-my-profile-toolbar []
```
Comment (yenda) [inline src/status_im/profile/edit/views.cljs]: I assumed it was a typo was it the case @rasom ?

**Comment by flexsurfer** [comment]
@yenda something weird with the indentations

File: src/status_im/ui/screens/profile/validations.cljs [hunk 2]
```diff
@@ -0,0 +1,25 @@
+(ns status-im.ui.screens.profile.validations
```
Comment (flexsurfer) [inline src/status_im/ui/screens/profile/validations.cljs]: validations should be moved in db

File: src/status_im/ui/screens/profile/styles.cljs [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
+  (:require [status-im.components.styles
+             :refer
+             [color-black
+              color-gray4
+              color-gray5
+              color-light-blue
+              color-light-gray
+              color-white
+              text1-color]]
+            [status-im.utils.platform :as plateform])
```
Comment (flexsurfer) [inline src/status_im/ui/screens/profile/styles.cljs]: platform

File: src/status_im/ui/screens/profile/validations.cljs (hunk 2 above)
Comment (flexsurfer) [inline src/status_im/ui/screens/profile/validations.cljs]: ok, it's just first step

Comment (yenda) [inline src/status_im/ui/screens/profile/validations.cljs]: yes but you are right it is not a breaking change so I'll move it in this PR

File: src/status_im/ui/screens/profile/styles.cljs (hunk 3 above)
Comment (yenda) [inline src/status_im/ui/screens/profile/styles.cljs]: pardon my french :D

**Comment by asemiankevich** [comment]
I've tested both on iOS (0.9.9d135) and Andriod (0.9.8-135-g04bb99f8+)

Checks performed: profile editing, field/length validation for user name, viewing profiles via chat.

**Pull request merged**